projects/canopy/src/lib/header/header.component.html

```diff
@@ -1,7 +1,52 @@
-<a *ngIf="logoHref; else imgTpl" [attr.href]="logoHref" class="lg-header__link">
-  <ng-container *ngTemplateOutlet="imgTpl"></ng-container>
-</a>
+<div lgContainer>
+  <div lgRow>
+    <div lgCol="12">
+      <div class="lg-header__wrapper">
+        <div class="lg-header__logos-wrapper">
+          <ng-container
+            *ngTemplateOutlet="
+              logoHref ? linkTpl : imgTpl;
+              context: { img: { logoAlt: logo, logoHref: logoHref, logo: logo } }
```

Comment:
not 100% sure but is it possible to use the shorthand properties notation for the logo and logoHref within a ngTemplateOutlet?

Reply:
No, I tried and I couldn't :/